webapp-django/crashstats/crashstats/views.py

```diff
@@ -1050,14 +1049,23 @@ def crashes_per_user(request, default_context=None):
     default = request.session.get('date_range_type', 'report')
     context['date_range_type'] = params.get('date_range_type') or default
 
-    api = SuperSearchUnredacted()
-    # XXX HACK! Because of a bug in SuperSearch we can't search for
-    # '4X.0b' so we have to change that to '4X.0b1' :(
-    # See https://bugzilla.mozilla.org/show_bug.cgi?id=1197251
+    params['original_versions'] = params['versions'][:]  # make a copy
+
+    # explode those ending in 'b'
+    exploded_version_numbers = {}
     for i, version in enumerate(params['versions']):
         if version.endswith('b'):
-            params['versions'].insert(i, version + '1')
             params['versions'].remove(version)
+            # look through all the releases for this product
+            exploded_version_numbers[version] = [
+                x['version']
+                for x in context['releases'][params['product']]
+                if x['version'].startswith(version) and x['version'] != version
+            ]
+    for versions in exploded_version_numbers.values():
+        params['versions'].extend(versions)
+
```

Comment:
Most of this should not be needed. Super Search is doing that for you (as proved in your comments in the bug). You only need the part where you merge the versions together, but there is no need to split in the params.